add sockjs support (#24)
blackhorse-one/stomp_dart_client: README.md

| onDebugMessage: Function(String)             | Function to be called for debug messages generated by the internal message handler                         |

## Use Stomp with SockJS
Use StompConfig.SockJS constructor instead of default StompConfig constructor.

> The url must be start with http/https instead of ws/wss.

Reviewer: ```suggestion
The url has to start with http/https instead of ws/wss.
```

> The StompClient url become automatically a ws url with SOCKJs pattern url like wss://host:port/path/xxxx/yyyy/websocket

Reviewer: I think this line can be removed or should expand on what xxxx and yyyy are. This probably goes into to much protocol specific detail, so i suggest removing it.
